CMakeLists.txt

```diff
@@ -20,30 +20,35 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+include(GNUInstallDirs)
+
 add_subdirectory(media_driver)
 
 if (DEFINED BUILD_ALONG_WITH_CMRTLIB)
     add_subdirectory(cmrtlib)
 endif()
 
 if (TARGET iHD_drv_video AND TARGET igfxcmrt)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION /usr/lib/x86_64-linux-gnu/dri/)
-    install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh  DESTINATION /etc/profile.d/)
-    install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION /usr/lib/x86_64-linux-gnu/)
+
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh.in ${CMAKE_CURRENT_SOURCE_DIR}/media_driver/cmake/linux/intel-media.sh)
```

Comment:
hmm... this is an outdated change.  latest changes use BINARY instead of SOURCE.  Perhaps your comments got cached before I made the change and only got posted after you approved the latest.

Reply:
don't know what happened. but when merging, it should use the latest version. Thanks!